llm_handler/__init___.py

```
@@ -0,0 +1,11 @@
+from .claude import send_to_claude
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)